vocabulary/do_not_capture.csv

```diff
@@ -0,0 +1,6 @@
+field_id,name,value,reason_std,reason,mapping_approach
```

Comment:
There already exists the file [ignore.csv](https://github.com/thehyve/ukbiobank-omop-etl/blob/master/vocabulary/ignore.csv).

Please add the rows of _ignore.csv_ in _do_not_capture.csv.